src/pb_stub.h

```diff
@@ -161,6 +161,30 @@ class LogMessage {
 
 #define LOG_FL(FN, LN, LVL) LogMessage((char*)(FN), LN, LVL).stream()
 
+
+class ModelScanner {
```

Comment:
Instead of a separate class - is it possible to have the state return the reconciled model path and then keep the rest of the stub setup the same (just based on the path that was returned based on the platform field in model state)?